docs/integrations/databases/opentelemetry/mongodb-opentelemetry.md

```diff
@@ -257,24 +277,54 @@ Use this dashboard to:
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/MongoDB-OpenTelemetry/MongoDB-Query-Logs.png' alt="Query Logs" />
 
+### Sharding
```

Comment:
Do we have a sharding dashboard ?

Reply:
Yes it is already there & it is logged based dashboard

Reply:
So was the doc missing the description for sharding dashboard ?